### DISCUSSION TITLE
teoria relazionale - 11

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#78] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/ea283b85-28c6-46b0-8018-bcf6f54ffdf2)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/dba7c4bd-a9f4-41ee-a33f-87121a1af9be)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-27 170438](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/864c07f6-39fa-42ff-a3d7-13c90c2abd05)
![Immagine 2024-01-27 170455](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/5002b782-fde0-4452-a592-05fa1c4429ac)
![Immagine 2024-01-27 170519](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/824a5cbc-9033-49ed-9f1b-28d71cc6bbea)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/66fdaac3-92e2-4ab6-8315-849aa9ee136c)
